lib/pf/pfcmd/checkup.pm

```diff
@@ -173,6 +173,7 @@ sub service_exists {
     my (@services) = @_;
 
     foreach my $service (@services) {
+        next if ($service eq 'pf');
```

Comment:
Does this remove the checkup when starting all PF services ?

If so, it doesn't sound that good since some people may miss out on major issues

Reply:
No it doesn't , here a sample when i restart pf:
```
fabian81:/usr/local/pf$ bin/pfcmd service pf restart
service|command
api-frontend|Service enabled
etcd|Service enabled
fingerbank-collector|Service enabled
haproxy-db|Service enabled
haproxy-portal|Service enabled
httpd.aaa|Service enabled
httpd.admin|Service enabled
httpd.collector|Service disabled
httpd.dispatcher|Service enabled
httpd.parking|Service enabled
httpd.portal|Service enabled
httpd.proxy|Service disabled
httpd.webservices|Service enabled
iptables|Service enabled
keepalived|Service enabled
netdata|Service enabled
pfbandwidthd|Service disabled
pfdetect|Service disabled
pfdhcp|Service enabled
pfdhcplistener|Service enabled
pfdns|Service enabled
pffilter|Service enabled
pfipset|Service enabled
pfmon|Service enabled
pfperl-api|Service enabled
pfqueue|Service enabled
pfsso|Service enabled
pfstats|Service enabled
radiusd-eduroam|Service enabled
radiusd-acct|Service enabled
radiusd-auth|Service enabled
radiusd-load_balancer|Service enabled
radiusd-cli|Service enabled
radsniff|Service enabled
redis_ntlm_cache|Service disabled
redis_queue|Service enabled
routes|Service disabled
snmptrapd|Service disabled
tc|Service enabled
winbindd|Service enabled
service|command
haproxy-db|stop
pf|stop
Could not write namespace config::PfDefault to L2 cache !
....
Could not write namespace resource::unified_api_system_user to L2 cache !
haproxy-db|start
Checking configuration sanity...
WARNING - template directory '/usr/local/pf/html/captive-portal/profile-templates/Wire' for profile Wire does not exist using default templates
WARNING - template directory '/usr/local/pf/html/captive-portal/profile-templates/Dot1x' for profile Dot1x does not exist using default templates
pf|start
```

Reply:
As they said in France:
"Superbe! maintenant allons chercher une baguette chez le boulanger"